rust/src/store.rs

```diff
@@ -1369,6 +1398,26 @@ impl CommandStore for IndexerStore {
                     .and_then(|s| s.parse().ok())
             }))
     }
+
+    fn get_internal_commands_interator(&self, mode: speedb::IteratorMode) -> DBIterator<'_> {
+        self.database.iterator_cf(self.internal_commands_cf(), mode)
+    }
+}
+
+fn compare_feetransfer_keys(a: &[u8], b: &[u8]) -> std::cmp::Ordering {
+    let str_a = std::str::from_utf8(a).expect("Valid UTF-8");
+    let str_b = std::str::from_utf8(b).expect("Valid UTF-8");
+
+    fn extract_block_height(s: &str) -> u32 {
+        let start = s.find('-').expect("Missing first '-' delimiter") + 1;
+        let end = s[start..].find('-').expect("Missing second '-' delimiter") + start;
+        s[start..end].parse::<u32>().expect("Valid block height")
+    }
```

Comment:
It feels like we could probably reuse this function elsewhere but it is certainly fine here.